tests/fullstack-test/expr/unhex.test

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022 PingCAP, Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+mysql> drop table if exists test.t;
+mysql> create table if not exists test.t(a char(100), b int);
+
+mysql> insert into test.t values('7777772E70696E676361702E636F6D', 3039);
+mysql> insert into test.t values('61626364', 61626364);
+mysql> insert into test.t values('GG', -1);
+mysql> insert into test.t values('E38195E38289E381ABE585A5', 313233);
+mysql> insert into test.t values('F09F8DBB', 414243);
+mysql> alter table test.t set tiflash replica 1;
```

Comment:
Just a suggestion, maybe you could add more character set tests like this，
https://github.com/pingcap/tiflash/blob/301561b850ca3545ae359b5430321cee3523f90c/dbms/src/Functions/tests/gtest_strings_upper.cpp#L60. Others LGTM.